### DISCUSSION TITLE
Does BRIDGE required ?

### DISCUSSION THREAD

**Discussion by Stoufiler** [mvdwetering/huesyncbox#42] [Q&A]
Hi !

Thanks for the integration!

I wanted to know if it is possible to use this integration without the Hue BRIDGE?

Indeed, I got rid of it in favor of a ZigBee DeconZ gateway

Also my lights are already connected to DeconZ

Is it possible to add the Sync BOX in DeconZ and have the integration communicate with it through this?

Thanks

**Comment by mvdwetering** [comment]
This integration controls the [Philips Hue Play HDMI Sync Box](https://www.philips-hue.com/en-gb/p/hue-play-hdmi-sync-box-/8718699704803), not a Philips Hue bridge.

The Philips Hue Play HDMI Sync Box requires a Philips Hue bridge to control the lights, it will not work without a bridge.

**Reply by Stoufiler** [reply]
Yes I understand !

But i've already have philips hue lights bulbs and I removed the bridge to use Deconz instead

My question was : Is it possible to control the HDMI Sync Box without the HUE Bridge

**Reply by mvdwetering** [reply]
No, the Philips Hue HDMI Sync Box requires a Philips Hue bridge.

**Reply by Stoufiler** [reply]
Oh damn

I don't want to use the Philips Bridge :/

Thanks for reply
